src/avif.c

```diff
@@ -116,7 +116,7 @@ const char * avifProgressiveStateToString(avifProgressiveState progressiveState)
     return "Unknown";
 }
 
-// This function assumes nothing in this struct needs to be freed; use avifImageClear() externally
+// This function assumes nothing in this struct needs to be freed.
```

Comment:
Joe: I can also remove this comment completely. Would you prefer that?

Reply:
Sure, just remove the whole comment.

Reply:
Done.